.travis.yml

```diff
@@ -10,12 +10,14 @@ cache:
     - $HOME/.composer/cache
 
 php:
+  - 7.3
   - 7.2
   - 7.1
   - 7.0
 
 env:
   - WP_VERSION=latest WC_VERSION=latest
+  - WP_VERSION=latest WC_VERSION=3.5.2
```

Comment:
These environments need to be fully updated - Post 5.0, WC version needs to be 3.5.1 or higher.

So the WP_VERSION should be 4.9.9 and the mix of the WC_VERSIONs listed should only apply to that version.

Reply:
See, this is why I have you review these things. Will get this updated, thank you 😄

Reply:
Also of note is that 7.3 is only supported in WP 5.0+.